pkg/api/types.go

```diff
@@ -1993,6 +1993,8 @@ const (
 	NodeOutOfDisk NodeConditionType = "OutOfDisk"
 	// NodeMemoryPressure means the kubelet is under pressure due to insufficient available memory.
 	NodeMemoryPressure NodeConditionType = "MemoryPressure"
+	// NodeNetworkingReady means that network for the node is correctly configured.
+	NodeNetworkingReady NodeConditionType = "NetworkingReady"
```

Comment:
+1. Why not invert the condition to be `NetworkingNotReady`, set it to `true` by default when kubelet registers a node, and have the route controller set that condition to `false` later?